arm-advisor/2016-07-12-preview/swagger/advisor.json

```diff
@@ -0,0 +1,461 @@
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/microsoft.Advisor/generaterecommendations/{operationId}": {
+			"get": {
+				"tags": ["GenerateRecommendations"],
+				"operationId": "GenerateRecommendations_GET",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "operationId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "api-version",
+					"in": "query",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"type": "object"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/subscriptions/{subscriptionId}/providers/microsoft.Advisor/recommendations": {
+			"get": {
+				"tags": ["GetRecommendations"],
+				"operationId": "GetRecommendations_GetRecomendation",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [{
+					"name": "subscriptionId",
+					"in": "path",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				},
+				{
+					"name": "recomendationId",
+					"in": "query",
+					"required": true,
+					"type": "string",
+					"format": "uuid"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationDetail"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/{recomendationId}": {
+			"get": {
+				"tags": ["GetRecommendations"],
+				"operationId": "GetRecommendations_GetRecomendation",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"parameters": [
+				{
+					"name": "recomendationId",
+					"in": "path",
+					"required": true,
+					"type": "string"
+				}],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/ResourceRecommendationDetail"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/providers/microsoft.Advisor/operations": {
+			"get": {
+				"tags": ["OperationDiscovery"],
+				"operationId": "OperationDiscovery_GetOperationsForDiscovery",
+				"consumes": [],
+				"produces": ["application/json",
+				"text/json"],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/CollectionResponse[OperationEntity]"
+						}
+					}
+				},
+				"deprecated": false
+			}
+		},
+		"/{recomendationId}/suppressions/{name}": {
+			"put": {
+				"tags": ["Suppression"],
+				"operationId": "Suppression_Suppressions",
```

Comment:
P0: Per the Noun_Verb convention for Operation Ids, the noun 'Suppression' should not appear after the underscore.

If you do not update this at this time, all the naming changes that you potentially do will break an SDK and that is why we recommend doing this change now.